x/upgrade/client/cli/tx.go

```diff
@@ -0,0 +1,184 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	upgrade "github.com/cosmos/cosmos-sdk/x/upgrade/internal/types"
+)
+
+const (
+	// TimeFormat specifies ISO UTC format for submitting the upgrade-time for a new upgrade proposal
+	TimeFormat = "2006-01-02T15:04:05Z"
+
+	FlagUpgradeName   = "upgrade-name"
```

Comment:
I'm curious if the `upgrade-` is redundant? Thoughts?

Reply:
I think `--name` is used as a flag to select the key already.
Open for shorter names though

Reply:
`upgrade-id`?

Reply:
I think we should move this flag to an argument as it is required in all cases when running this command.

Reply:
sounds good to me